src/net/cgrand/enlive_html.clj

```diff
@@ -992,7 +992,7 @@
 (defn- nodify [node-spec]
   (cond
     (string? node-spec) node-spec
-    (vector? node-spec)
+    (and (vector? node-spec) (not (map? (first node-spec)))) 
```

Comment:
Rather than `(not (map? (first node-spec)))`, `(not (keyword? (first node-spec)))` seems to better handle cases such as `["abc" {:tag :b :content ["def"]} "ghi"]`. Doesn't it?